datalad/interface/rerun.py

```diff
@@ -360,9 +372,41 @@ def _rerun(dset, results):
                 None, ["git", "cherry-pick", res["commit"]],
                 check_fake_dates=True)
             yield res
+        elif rerun_action == "merge":
+            # We uniquify because content may not change (depending on the
+            # value of --onto).
+            new_parents = unique(
+                filter(None, (new_bases.get(p) for p in res["parents"])))
+            if len(new_parents) < 2:
+                lgr.warning(
+                    "Skipping merge %s because only one parent "
+                    "is in the specified revision range",
+                    res["commit"])
```

Comment:
Hm, if I was to specify a range for rerunning - don't we still want to merge previous histories (as is, without rerunning) if that was done on original history?

Reply:
Good question.  Maybe?  Another thing I need to think about more.